docs/source/shared/add-query.mdx

```diff
@@ -0,0 +1,18 @@
+1. Create a directory for your GraphQL files: `src/main/graphql/com/example/`
```

Comment:
These shared files are not included in the index, is that intentional?

Reply:
Yep, they are shared between the Java/Kotlin/Multiplatform pages thanks to the `.mdx` magic

Reply:
Today I learned.